### ISSUE TITLE
ctrl-p still launching fuzzy-finder:toggle-file-finder

### ISSUE THREAD

**Issue by turnipcyberveggie** [mndrix/atom-emacs-core-keys#6]
ctrl-p was launching the fuzzy-finder:toggle-file-finder dialog. 

I fixed it by updating keymap.cson:
'.platform-win32': 'ctrl-p': 'core:move-up'
'.platform-darwin, .platform-win32, .platform-linux': 'ctrl-p': 'core:move-up'

**Comment by mndrix** [comment]
I'm not seeing this problem locally.  What OS are you on?  What version of Atom?  What version of emacs-core-keys?

Can you enable the "Key Binding Resolver" in Atom, press `ctrl-p` and attach a screenshot of the resolver's window?

Thanks.

**Comment by turnipcyberveggie** [comment]
Windows 10 - 1607
Atom 1.12.3
![emacs_atom_versoin](https://cloud.githubusercontent.com/assets/5711969/20401878/835fbf32-acc8-11e6-9196-5c56365be991.png)
![emcac_ctrlp_keyresolover](https://cloud.githubusercontent.com/assets/5711969/20401884/861c8e80-acc8-11e6-8536-7d5183e9c347.png)

**Comment by mndrix** [comment]
This problem occurs because both emacs-core-keys and fuzzy-finder packages bind `ctrl-p` with identical [specificity](http://flight-manual.atom.io/behind-atom/sections/keymaps-in-depth/#specificity-and-cascade-order).  In that case whichever package loads last "wins" and has its keybinding honored while the other is ignored.  As far as I can tell, package load order is not defined (it would be nice if packages installed by the user took precedence over system packages).

Anyway, to resolve this at the package level, we could add a keybinding for `ctrl-p` with higher specificity than the fuzzy-finder keybinding.  Maybe something like:

``` cson
'body.platform-darwin, body.platform-linux, body.platform-win32':
    'ctrl-p': 'core:move-up'
```

I need to read Atom's [specificity calculation code](https://github.com/atom/clear-cut/blob/ba12f746679ce45a267c73283a31fbe8eb6fdb39/index.js) to find a selector that has a higher specificity but still matches just as generally as the one we're using now.

As a temporary workaround, anyone else that encounters this problem can follow the lead of @turnipcyberveggie and add a `ctrl-p` binding to their own private keymap.cson.  Since [#12760](https://github.com/atom/atom/pull/12760) was merged in [Atom 1.12.0](https://github.com/atom/atom/releases/tag/v1.12.0) a user's bindings always take precedence over a package's bindings.

**Comment by mndrix** [comment]
I verified that prepending `body` to the selector makes it slightly more specific than the rule in fuzzy-finder. In the Atom console:

```
> calc = require('clear-cut').calculateSpecificity
> calc(".platform-win32")
10
> calc("body.platform-win32")
11
```

Changed in commit fae4665.  Published as v0.2.3.  Please let me know if this doesn't fix it for you @turnipcyberveggie
